patra_toolkit/patra_model_card.py

```diff
@@ -314,14 +314,14 @@ def submit(self, patra_server_url):
         if self.validate():
             try:
                 self.id = self._get_hash_id(patra_server_url)
+
                 patra_submit_url = f"{patra_server_url}/upload_mc"
                 headers = {'Content-Type': 'application/json'}
                 response = requests.post(patra_submit_url, json=json.loads(str(self)), headers=headers)
                 response.raise_for_status()
                 return response.json()
-            except requests.exceptions.RequestException as e:
-                print("The Patra Server cannot be reached. Please try again.")
-                return None
+            except ValueError as e:
+                raise e
```

Comment:
Don't just raise the error.Have message associated with it to know where it came from.
Ex:
except ValueError as e:
    raise ValueError("This is what happened...") from e